dashboard/src/components/Filter/CodeBlock.tsx

```diff
@@ -21,10 +32,130 @@ interface IHighlightedCode {
   errorCount: number;
 }
 
+interface CodeComponentProps {
+  fullscreen?: boolean;
+}
+
+const CodeBlockDialog = ({
+  CodeComponent,
+}: {
+  CodeComponent: ({ fullscreen }: CodeComponentProps) => JSX.Element;
+}): JSX.Element => {
+  return (
+    <div className="relative">
+      <Dialog>
+        <DialogTrigger asChild className="p-2">
+          <Button
+            variant="outline"
+            className="absolute right-[10px] top-[10px] opacity-50 hover:opacity-100"
+          >
+            <GoScreenFull size={25} />
+          </Button>
+        </DialogTrigger>
+
+        <CodeComponent />
+
+        <DialogContent className="sm:max-w-[70%]">
+          <DialogHeader>
+            <DialogTitle>
+              <FormattedMessage id="global.logs" />
+            </DialogTitle>
+          </DialogHeader>
+          <CodeComponent fullscreen />
+        </DialogContent>
+      </Dialog>
+    </div>
+  );
+};
+
+const Code = ({
+  highlightedCode,
+  className,
+  footer,
+}: {
+  highlightedCode: string;
+  className?: string;
+  footer: JSX.Element;
+}): JSX.Element => {
+  return (
+    <>
+      <pre
+        className={cn(
+          'w-full max-w-[calc(100vw_-_398px)] overflow-x-auto rounded-md bg-[#DDDDDD] p-4 font-mono text-[#767676]',
```

Comment:
Put those colors in `tailwind.config.js`

Reply:
we are not putting colors on tailwind config because it is not consistent on figma

Reply:
I understand  but IMO we still should use a  schema for colors